### DISCUSSION TITLE
Where do I find PK-Sim Example files that can be downloaded

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1613] [Questions & Problems]
**IMPORTANT: Issues are now obsolete**
Starting May 7th, 2021, the OSP issues will not be used anymore for discussions. Instead, the Discussions feature from Github will be used

Please see the announcement here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/894

To start a discussion, simply go to 
https://github.com/Open-Systems-Pharmacology/Forum/discussions and click on New Discussion

**Comment by Yuri05** [comment]
Examples can be found under https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library

**Reply by zt1947** [reply]
Thanks!
